docs/configure-rest.md

```diff
@@ -185,6 +185,78 @@ exporters, the following parameters can be defined:
 * `instance_labels` (list): display names of labels to export as a separate data-point
 * `include_all_labels` (bool): export all labels with each data-point (overrides previous two parameters)
 
+#### Endpoints
+
+In Harvest REST templates, `endpoints` are additional queries that enhance the data collected from the main query. The main query, identified by the `query` parameter, is the primary REST API for data collection. For example, the main query for a `disk` object is `api/storage/disks`.
```

Comment:
after the first sentence, maybe add this:
Typically `endpoints` are used to query the private CLI to add metrics that are not available via ONTAP's public REST API.